staging/prometheus-operator/Chart.lock

```diff
@@ -0,0 +1,12 @@
+dependencies:
+- name: kube-state-metrics
+  repository: https://charts.helm.sh/stable
+  version: 2.9.4
+- name: prometheus-node-exporter
+  repository: https://prometheus-community.github.io/helm-charts
+  version: 1.12.0
+- name: grafana
+  repository: https://grafana.github.io/helm-charts
+  version: 6.1.17
```

Comment:
upstream (on 12.11.3) had this at `6.1.16`, any reason why we updated it here? I mostly just prefer to stay as close to upstream as possible

Reply:
its a patch bump, i think we are ok, but i can revert it if you feel strongly about it.

Reply:
nah I'm cool with it, we'll probably do a bump soon-ish anyways since newer chart versions have already come out